pkgs/os-specific/linux/wooting-xinput-udev-rules/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv }:
+
+stdenv.mkDerivation rec {
+  pname = "wooting-xinput-udev-rules";
+  version = "20201129";
+
+  # Source: https://wooting.helpscoutdocs.com/article/83-guide-configuring-xinput-support-for-the-wootings-under-linux
+  src = [ ./wooting-xinput.rules ];
```

Comment:
If you already vendored the text, you could probably use a trivial builder like `writeTextFile`

Reply:
I've left the rules in a separate file for now, and have added some new rules that were added (for new keyboard models) since this PR was made.